### PR TITLE
Update recipe extractor to grab facilities and tool ids

### PR DESCRIPTION
**Pull request by mustyoshi**
- In the wiki pages it's spelled facilities rather than facility
- Also added wiki-data folder to gitignore
- Started looking up the tool name to get the tool ids

### CODE REVIEW

**Comment by mustyoshi** [comment]
@AntonyGarand Take a look, lmk if there's anything you'd want me to change about the PR.

File: src/modules/extractors/recipes.extractor.ts [hunk 1]
```diff
@@ -211,7 +211,13 @@ export class RecipesExtractor {
     const ticks = isNaN(Number(recipeProperties.ticks))
       ? null
       : Number(recipeProperties.ticks);
-
+    let toolIds: number[] = [];
+    if (recipeProperties.tools) {
+      toolIds = recipeProperties.tools.split(',').map((v) => {
+        const item = this.itemExtractor.getItemByName(v);
+        return item?.id;
+      });
```
Comment (AntonyGarand) [inline src/modules/extractors/recipes.extractor.ts]: Should filter out the empty values to ensure the list isn't a bunch of empty entries

File: src/modules/extractors/recipes.extractor.ts [hunk 2]
```diff
@@ -211,7 +211,13 @@ export class RecipesExtractor {
     const ticks = isNaN(Number(recipeProperties.ticks))
       ? null
       : Number(recipeProperties.ticks);
-
+    let toolIds: number[] = [];
```
Comment (AntonyGarand) [inline src/modules/extractors/recipes.extractor.ts:214]: can use const

**Comment by AntonyGarand** [comment]
Thanks for the PR! Nice catch with the facilities, lgtm
Left a minor comment but it'll be merge once fixed

File: src/modules/extractors/recipes.extractor.ts (hunk 2 above)
Comment (mustyoshi) [reply src/modules/extractors/recipes.extractor.ts:214]: We assign to it in the if statement, so it needs to be mutable.

Comment (AntonyGarand) [reply src/modules/extractors/recipes.extractor.ts:214]: Ah true, nevermind 🤦‍♂️

**Comment by mustyoshi** [comment]
I have a follow up PR that is a little bit more dubious after this one.

**Review by AntonyGarand — APPROVED** [review]

**Pull request merged**